packages/gafl-webapp-service/src/handlers/page-handler.js

```diff
@@ -116,6 +133,8 @@ export default (path, view, completion, getData) => ({
 
     pageData.displayAnalytics = displayAnalytics(request)
 
+    await skipPage(request)
```

Comment:
I don't think you're testing for this. Do any tests fail when you remove it?

Also, I don't think `skipPage` is a specific enough name - there's nothing to indicate that it's about analytics. Anyone reading this will wonder what skipPage is doing. `omitPageFromAnalytics` might be a better name?

Reply:
Yes it does look like several tests fail without this function being used. 

Good point, that name is much more descriptive will use it instead thanks.